quickstarts/kubernetes-prometheus/config.yml

```diff
@@ -0,0 +1,44 @@
+slug: kubernetes-prometheus
+title: Kubernetes (Prometheus)
+description: |
+  ## What is Kubernetes?
+  Kubernetes is an open-source system for automating deployment, scaling, and, management of containerized applications. The New Relic Kubernetes (Prometheus) monitoring quickstart utilizes Prometheus to give you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
+  
+  ## What is Prometheus? 
+  Prometheus is an open-source monitoring and alerting toolkit. 
+
+  ### Kubernetes (Prometheus) quickstart highlights
+  The New Relic Kubernetes quickstart uses dashboards to proactively monitor your metrics, like:
+  - memory and cpu utilization 
+  - running pods and containers 
+  - kubelet pod start metrics 
+  - compute resources by pod, node, cluster, and namespace and more. 
+summary: |
+  Monitoring Kubernetes is crucial to gain instant visibility into Kubernetes clusters and workloads. 
+  Prometheus' ability to monitor a wide variety of data necessitates monitoring the toolkit itself. New Relic helps store, manage, and view telemetry data from your Prometheus setup, relieving you of a significant operational burden.
+  Download New Relic Kubernetes (Prometheus) quickstart to proactively monitor Kubernetes cluster health and capacity.
+level: Community
+authors:
+  - New Relic 
+documentation:
+  - name: Kubernetes Prometheus integration docs
+    description: |
+     New Relic's Prometheus agent is a thin wrapper on top of Prometheus server to run it in agent mode. With this integration, you can create a YAML configuration file used by Prometheus that's ready to send metrics to New Relic.
```

Comment:
Can we add instructions to follow the README in @htroisi's changes? 


The default docs won't have all the info. For customers that already have the configurator: We should specify that the default installation of the configurator will be missing some of the metrics needed for these dashboards.

Reply:
Will that PR be merged or will it just be a pr? I think we would prefer to link the actual MD instead of the PR w/ the examples?

Reply:
That PR should be merged before this one, agree we should link to the file itself rather than a PR